pkg/tests/handlers/accounts.go

```diff
@@ -226,12 +227,23 @@ func TestClusterAuthorizations(ctx context.Context, options *types.TestOptions)
 	return nil
 }
 
+func randomizeResourceName(ResourceName_list [5]string) string {
+	ResourceName := fmt.Sprint(ResourceName_list[rand.Intn(len(ResourceName_list))])
```

Comment:
`ResourceName_list` is already a slice of strings, there is no need to convert it to string again.
Also variable naming, this should not start with Capital `R`